### PR TITLE
"Play" Arrow on VideoCards

### PR DESCRIPTION
**Pull request by mwilkins74**
Implemented this just in case we _do_ want to run with it. 

Screen Shot 2024-03-15 at 2 15 04 PM

### CODE REVIEW

**Comment by ChuckBTaylor** [comment]
Let's run this by Vika, but otherwise looks great

File: src/components/molecules/videoCard/VideoCard.tsx [hunk 1]
```diff
@@ -48,7 +50,29 @@ export function VideoCard({ video, pageType }: VideoCardProps) {
...
+          <BiSolidRightArrow
+            style={{
+              position: 'absolute',
+              top: '50%',
+              left: '50%',
+              transform: 'translate(-50%, -50%)',
+              fontSize: '30px',
+              color: '#303030',
+            }}
+          />
```
Comment (ChuckBTaylor) [inline src/components/molecules/videoCard/VideoCard.tsx:74]: 🔧 Just based on the image, Vika's play button is a bit more angular. 

Can we do `GrPlayFill` or `IoPlaySharp`?

Comment (mwilkins74) [reply src/components/molecules/videoCard/VideoCard.tsx:74]: This is `GrPlayFill`:
Screen Shot 2024-04-03 at 2 56 30 PM

This is `IoPlaySharp`:
Screen Shot 2024-04-03 at 2 57 53 PM

It looks closer to `GrPlayFill` to me, but let me know on preference!

Comment (ChuckBTaylor) [reply src/components/molecules/videoCard/VideoCard.tsx:74]: Probably `GrPlayFill`

**Review by ChuckBTaylor — APPROVED** [review]

**Pull request merged**